console/src/components/Services/Data/TablePermissions/Permissions.js

```diff
@@ -528,6 +556,60 @@ class Permissions extends Component {
       return _upsertSection;
     };
 
+    const getAggregationSection = permsState => {
+      let _aggregationSection = '';
+      const query = permsState.query;
+
+      if (query === 'select') {
+        const dispatchToggleAllowAggregation = checked => {
+          dispatch(permToggleAllowAggregation(checked));
+        };
+
+        const aggregationAllowed = permsState.select
+          ? permsState.select.allow_aggregations
+          : false;
+
+        // Aggregation Tooltip
+        const aggregationToolTip = (
+          <Tooltip id="tooltip-aggregation">
+            Allow queries with aggregate functions like sum, count, avg, max,
+            min, etc
+          </Tooltip>
+        );
+
+        // TODO: Fix the controlled component
+        _aggregationSection = (
+          <div>
```

Comment:
Can you align this div with the `columns`

![screenshot 559](https://user-images.githubusercontent.com/25095884/47430568-677c8180-d7b7-11e8-9c65-757ea8f2c006.jpg)


@surendran82 can you help arvi?